lighthouse-core/lib/i18n/i18n.js

```diff
@@ -180,36 +188,40 @@ function _collectAllCustomElementsFromICU(icuElements, seenElementsById = new Ma
  * will apply Lighthouse custom formatting to the values based on the argumentElement
  * format style.
  *
- * @param {Array<import('intl-messageformat-parser').Element>} argumentElements
+ * @param {string} icuMessage
```

Comment:
included for logging. While playing around with some other error conditions I found it basically impossible to know *which* message the erroring `itemCount` or `timeInMs` was coming from, so this was really needed in some form